org-formation/040-budgets/_tasks.yaml

```diff
@@ -12,3 +12,12 @@ BudgetAlarms:
     budgetName: !Sub '${resourcePrefix}-budget-${CurrentAccount.AccountId}'
     budgetAmount: !GetAtt CurrentAccount.Tags.budget-alarm-threshold
     emailAddress: !GetAtt CurrentAccount.Tags.budget-alarm-threshold-email-recipient
+
+CostAndUsageReports:
+  Type: update-stacks
+  Template: cur.yaml
+  StackName: !Sub '${resourcePrefix}-monthly-cost-report'
```

Comment:
do you think this stack is going to contain only monthly reports or other frequencys?  if you think there will be other frequencies then i suggest a more generic name like `${resourcePrefix}-cost-report`.  Otherwise if this is going to be exclusively monthly reports then keep this name and make the resource name a little more specific, like `MonthlyCostAndUsageReports`.

Reply:
Ah, good point, if we do add more reports they would be in the same stack, so the stack shouldn't refer to the report frequency.